website/content/en/docs/deployment/cognito/manifest/guide-automated.md

```diff
@@ -164,8 +95,108 @@ make delete-kubeflow INSTALLATION_OPTION=helm DEPLOYMENT_OPTION=cognito
+
+* Create a user in Cognito user pool.
+* Create a profile for the user from the user pool.
+* Connect to the central dashboard.
+
+## Uninstall Kubeflow
+
+> Note: Delete all the resources you might have created in your profile namespaces before running these steps.
+
+1. Run the following commands to delete the profiles, ingress and corresponding ingress managed load balancer
+   ```bash
+    kubectl delete profiles --all
+    ```
+
+1. Delete the kubeflow deployment
+
+    > Note: Make sure you have the correct INSTALLATION_OPTION and DEPLOYMENT_OPTION environment variables set for your chosen installation.
+    {{< tabpane persistLang=false >}}
+    {{< tab header="Kustomize" lang="toml" >}}
+make delete-kubeflow INSTALLATION_OPTION=kustomize DEPLOYMENT_OPTION=cognito
+    {{< /tab >}}
+    {{< tab header="Helm" lang="yaml" >}}
+make delete-kubeflow INSTALLATION_OPTION=helm DEPLOYMENT_OPTION=cognito
+    {{< /tab >}}
+    {{< /tabpane >}}
+ 
+1. To delete the rest of the resources (subdomain, certificates etc.), run the following commands from the root of your repository:
+
+    * Find the configuration file `tests/e2e/utils/cognito_bootstrap/config.yaml` updated by the `cognito_post_deployment.py` script. If you did not use the script, update the name, ARN, or ID of the resources that you created in a separate yaml file in `tests/e2e/utils/cognito_bootstrap/config.yaml` by referring to the following sample:
+
+        ```yaml
+        # Sample config file
+        cognitoUserpool:
+            ARN: arn:aws:cognito-idp:us-west-2:123456789012:userpool/us-west-2_yasI9dbxF
+            appClientId: 5jmk7ljl2a74jk3n0a0fvj3l31
+            domainAliasTarget: xxxxxxxxxx.cloudfront.net
+            domain: auth.platform.example.com
+            name: kubeflow-users
+        kubeflow:
+            alb:
+                serviceAccount:
+                    name: alb-ingress-controller
+                    namespace: kubeflow
+                    policyArn: arn:aws:iam::123456789012:policy/alb_ingress_controller_kube-eks-clusterxxx
+        cluster:  
+            name: kube-eks-cluster
+            region: us-west-2
+        route53:
+            rootDomain:
+                certARN: arn:aws:acm:us-east-1:123456789012:certificate/9d8c4bbc-3b02-4a48-8c7d-d91441c6e5af
+                hostedZoneId: XXXXX
+                name: example.com
+            subDomain:
+                us-west-2-certARN: arn:aws:acm:us-west-2:123456789012:certificate/d1d7b641c238-4bc7-f525-b7bf-373cc726
+                hostedZoneId: XXXXX
+                name: platform.example.com
+                us-east-1-certARN: arn:aws:acm:us-east-1:123456789012:certificate/373cc726-f525-4bc7-b7bf-d1d7b641c238
+        ```
+    * Run the following command to install the script dependencies and delete the resources:
+    
+    > Note: You can rerun the script in case some resources fail to delete.
```

Comment:
needs a new line after note otherwise does not render correctly

Reply:
Struggling with the rendering of the Note in this bullet point section. Removing the `> Note:` part.